core/schema/container.go

```diff
@@ -714,6 +774,7 @@ type containerExecArgs struct {
 	// If the container has an entrypoint, ignore it for this exec rather than
 	// calling it with args
 	SkipEntrypoint *bool `default:"true"`
+	Expand         bool  `default:"false"`
```

Comment:
Move this into `core.ContainerExecOpts`, that way we don't duplicate between the legacy/new version of this.

Reply:
do we need to add this to useExecLegacy? I thought we don;t need to add new abilities to that function as its supported with only older version of engines?

Reply:
nvm, I see this is useful even if we are not making change to useExecLegacy. Moved the option to `core.ContainerExecOpts`.